Updated styles and layout to be compatible with twitter-bootstrap 4

armezit opened the pull request.

armezit committed:
Updated styles and layout to be compatible with twitter-bootstrap 4

armezit commented:
@DonMartin76 It would be nice to have sass support to let customizing bootstrap variables . Do you have any idea? I'll take a look at kickstarter and portal-env projects to find out possible approaches.

armezit commented:
It makes sense. I would implement it in just a couple of days or maybe even sooner.

DonMartin76 commented:
This will probably not make it into the 1.0 release, but I see it as a 1.1. There are still a couple of open questions we need to assess before we can really take it into the roadmap.

Over the next couple of days, I will be dropping the `wicked_1_0` and merge everything to `next`; can you rebase your PR to that branch after it has happened (i.e. *not yet*)? That would be great.

I will open an issue for this to make it more public what we are talking about.

armezit commented:
Sure. I'll rebase it whenever the `next` branch is ready. 
Also, I'll add support for compiling sass files in `portal-kickstarter` and it would be ready for more review when you open an issue.

DonMartin76 committed:
Update bootstrap to 3.4.0

DonMartin76 commented:
Branch `next` is up to date.

DonMartin76 committed:
Remove redis password definition; does not work.

-Wire to portal.com instead of portal.local, otherwise Google locally won't work

armezit committed:
Added SCSS files to enable customizing colors of portal ui

armezit committed:
Added SCSS files to enable customizing colors of portal ui

armezit committed:
Merge remote-tracking branch 'origin/twitter-bootstrap-4' into twitter-bootstrap-4

# Conflicts:
#	initial-config/static/content/stylesheets/style.scss
